### ISSUE TITLE
System.Xml not imported in TE3

### ISSUE THREAD

**Issue by m-kovalsky** [TabularEditor/TabularEditor3#539]
It appears that the System.Xml assembly is not imported automatically into TE3 (while it is in TE2). Running a c# script gives this error. I can resolve this by adding an assembly reference at the top of the script but it would be easier if TE2 & TE3 aligned with imported assemblies. Thanks!

![image](https://user-images.githubusercontent.com/29556918/168852496-875a58cb-660c-4930-8776-bfd4388e3b89.png)

**Comment by otykier** [comment]
Right, we have been removing assemblies that are not commonly used from the default imports in TE3. We will probably do the same thing in TE2 at some point to keep the scripting engines in sync between the two versions of TE, so the best solution is to add the `#r "System.Xml"` line at the top of any scripts that rely on this assembly.

**Comment by m-kovalsky** [comment]
Got it. Thanks for the update! Would you be sure to update the documentation so we know which assemblies are inherent in TE2 & TE3? Thanks!
